File: 02_variable/c/hello_variable.c
#include <stdio.h>

int main(void) {
    char name[] = "yoshikazu";
    int age = 48;
    double height = 170.5;

    printf("%s\n", name);
    printf("%d\n", age);
    printf("%f\n", height);

    printf("Hello, %s\n", name);
    printf("Age: %d\n", age);
    printf("Height: %.1f\n", height);

    return 0;
}

File: 02_variable/go/hello_variable.go
package main

import "fmt"

func main() {
	name := "yoshikazu"

	fmt.Println(name)
	fmt.Println("name")
	fmt.Println("Hello, " + name)

	fmt.Printf("Hello, %s\n", name)
	fmt.Println(fmt.Sprintf("Hello, %s", name))

	firstName, lastName := "yoshikazu", "ito"

	fmt.Println(firstName)
	fmt.Println(lastName)
	fmt.Printf("%s %s\n", lastName, firstName)
}
